Realm/ObjectServerTests/RLMUser+ObjectServerTests.mm

```diff
@@ -58,6 +58,36 @@ - (BOOL)waitForDownloadToFinish:(NSString *)partitionValue {
     return dispatch_semaphore_wait(sema, dispatch_time(DISPATCH_TIME_NOW, (int64_t)(timeout * NSEC_PER_SEC))) == 0;
 }
 
+- (BOOL)waitForUploadsForRealm:(RLMRealm *)realm {
```

Comment:
These are almost but not quite identical to the version in RLMSyncTestCase. I think the version there is functionally better (if the wait times out, this version silently continues without reporting an error), and the two should be unified rather than having the functionality in two places.